apps/webapp/shared/helper-card.tsx

```diff
@@ -0,0 +1,27 @@
+import { Card } from 'ui';
+import { ResponsiveImage } from './responsive-image';
+
+interface HelperCardProps {
+  src: string;
+  label: string;
+}
+
+export const HelperCard = ({ src, label }: HelperCardProps) => {
```

Comment:
Instead of helper, we should have something more descriptive. I think `LearnMoreCard` may be better. Or `EduInfoCard`.